### PR TITLE
docs(locale): pt_BR localization to builtin

### PR DESCRIPTION
**Pull request by YunaSatoy**
Addition of Brazilian Portuguese localization to builtin `builtin/locale/__builtin.pt_BR.tr`.

### CODE REVIEW

File: builtin/locale/__builtin.pt_BR.tr [hunk 1]
```diff
@@ -0,0 +1,246 @@
+# textdomain: __builtin
+Invalid parameters (see /help @1).=Parâmetros inválidos (veja /help @1).
+Too many arguments, try using just /help <command>=Muitos argumentos, tente usar apenas /help <comando>
+Available commands: @1=Comandos disponíveis: @1
+Use '/help <cmd>' to get more information, or '/help all' to list everything.=Use '/help <cmd>' para obter mais informações, ou '/help all' para listar tudo.
+Available commands:=Comandos disponíveis:
+Command not available: @1=Comando não disponível: @1
+[all | privs | <cmd>] [-t]=
```
Comment (srifqi) [inline builtin/locale/__builtin.pt_BR.tr:8]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.pt_BR.tr [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+Double-click to copy the entry to the chat history.=Clique duas vezes para copiar a entrada para o histórico do chat.
+Command: @1 @2=Comando: @1 @2
+Available commands: (see also: /help <cmd>)=Comandos disponíveis: (veja também: /help <cmd>)
+Close=Fechar
+Privilege=Privilégio
+Description=Descrição
+Empty command.=Comando vazio.
+Invalid command: @1=Comando inválido: @1
+Invalid command usage.=Uso de comando inválido.
+ (@1 s)=
```
Comment (srifqi) [inline builtin/locale/__builtin.pt_BR.tr:23]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.pt_BR.tr [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+Command execution took @1 s=A execução do comando levou @1 s
+You don't have permission to run this command (missing privileges: @1).=Você não tem permissão para executar este comando (privilégios faltando: @1).
+Unable to get position of player @1.=Incapaz de obter a posição do jogador @1.
+Incorrect area format. Expected: (x1,y1,z1) (x2,y2,z2)=Formato de área incorreto. Esperado: (x1,y1,z1) (x2,y2,z2)
+<action>=
```
Comment (srifqi) [inline builtin/locale/__builtin.pt_BR.tr:28]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.pt_BR.tr [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+Return list of all online players with privilege=Retornar lista de todos os jogadores online com privilégio
+Invalid parameters (see /help haspriv).=Parâmetros inválidos (veja /help haspriv).
+Unknown privilege!=Privilégio desconhecido!
+No online player has the "@1" privilege.=Nenhum jogador online tem o privilégio "@1".
+Players online with the "@1" privilege: @2=Jogadores online com o privilégio "@1": @2
+Your privileges are insufficient.=Seus privilégios são insuficientes.
+Your privileges are insufficient. '@1' only allows you to grant: @2=Seus privilégios são insuficientes. '@1' só permite que você conceda: @2
+Unknown privilege: @1=Privilégio desconhecido: @1
+@1 granted you privileges: @2=@1 concedeu-lhe privilégios: @2
+<name> (<privilege> [, <privilege2> [<...>]] | all)=
```
Comment (srifqi) [inline builtin/locale/__builtin.pt_BR.tr:48]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.pt_BR.tr [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+Give privileges to player=Conceder privilégios ao jogador
+Invalid parameters (see /help grant).=Parâmetros inválidos (veja /help grant).
+<privilege> [, <privilege2> [<...>]] | all=
```
Comment (srifqi) [inline builtin/locale/__builtin.pt_BR.tr:51]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.pt_BR.tr [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+Unhandled remove_player return code @1.=Código de retorno remove_player não tratado @1.
+Cannot teleport out of map bounds!=Não é possível teleportar fora dos limites do mapa!
+Cannot get player with name @1.=Não é possível obter jogador com o nome @1.
+Cannot teleport, @1 is attached to an object!=Não é possível teleportar, @1 está anexado a um objeto!
+Teleporting @1 to @2.=Teleportando @1 para @2.
+One does not teleport to oneself.=Não se teleporta para si mesmo.
+Cannot get teleportee with name @1.=Não é possível obter teleportado com o nome @1.
+Cannot get target player with name @1.=Não é possível obter jogador alvo com o nome @1.
+Teleporting @1 to @2 at @3.=Teleportando @1 para @2 em @3.
+<X>,<Y>,<Z> | <to_name> | <name> <X>,<Y>,<Z> | <name> <to_name>=
```
Comment (srifqi) [inline builtin/locale/__builtin.pt_BR.tr:88]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.pt_BR.tr [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+([-n] <name> <value>) | <name>=([-n] <nome> <valor>) | <nome>
+Set or read server configuration setting=Definir ou ler configuração do servidor
+Failed. Cannot modify secure settings. Edit the settings file manually.=Falhou. Não é possível modificar configurações seguras. Edite o arquivo de configurações manualmente.
+Failed. Use '/set -n <name> <value>' to create a new setting.=Falhou. Use '/set -n <nome> <valor>' para criar uma nova configuração.
+@1 @= @2=
+<not set>=<não definido>
+Invalid parameters (see /help set).=Parâmetros inválidos (veja /help set).
+Finished emerging @1 blocks in @2ms.=Finalizada a emergência de @1 blocos em @2ms.
+emergeblocks update: @1/@2 blocks emerged (@3%)=atualização de emergeblocks: @1/@2 blocos emergidos (@3%)
+(here [<radius>]) | (<pos1> <pos2>)=
```
Comment (srifqi) [inline builtin/locale/__builtin.pt_BR.tr:100]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.pt_BR.tr [hunk 8]
```diff
@@ -0,0 +1,246 @@
...
+Can use fly mode=Pode usar o modo voar
+Can use fast mode=Pode usar o modo rápido
+Can fly through solid nodes using noclip mode=Pode voar através de nós sólidos usando o modo noclip
+Can use the rollback functionality=Pode usar a funcionalidade de reversão
+Can enable wireframe=Pode ativar wireframe
+Unknown Item=Item desconhecido
+Air=Ar
+Ignore=Ignorar
+You can't place 'ignore' nodes!=Você não pode colocar nós 'ignorar'!
+print [<filter>] | dump [<filter>] | save [<format> [<filter>]] | reset=
```
Comment (srifqi) [inline builtin/locale/__builtin.pt_BR.tr:236]: This has not been translated. Even if it is the same, please also write it.

**Review by srifqi — CHANGES_REQUESTED** [review]
There are some issues.

**Comment by YunaSatoy** [comment]
> There are some issues.

Thanks for the review. I made the necessary corrections! :blush:

**Review by srifqi — COMMENTED** [review]
The translation is complete and no syntax error is found. (Oops, I chose the wrong type of review.)

**Review by srifqi — APPROVED** [review]
The translation is complete and no syntax error is found.

**Pull request merged**